kanidmd/src/lib/core/https/mod.rs

```diff
@@ -1163,6 +1164,140 @@ impl<State: Clone + Send + Sync + 'static> tide::listener::Listener<State> for T
 }
 */
 
+#[cfg(test)]
+mod tests {
```

Comment:
Tests should be at the bottom of the file.

Reply:
They totally will be once I'm done writing the code - just easier to keep it close and save epic-scrolling on my tiny laptop 😄